src/main/java/org/sagebionetworks/bridge/notification/worker/WorkerConfig.java

```diff
@@ -57,6 +77,51 @@ public void setExcludedDataGroupSet(Set<String> excludedDataGroupSet) {
         this.excludedDataGroupSet = excludedDataGroupSet != null ? excludedDataGroupSet : ImmutableSet.of();
```

Comment:
Do you want to use ImmutableSet.of(excludedDataGroupSet) to ensure the set is always immutable? Here and below.

Reply:
Fixed for all collection types in this class.